### PR TITLE
Code Structure Refactoring

### PR DESCRIPTION
**Pull request by tungle251**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,8 +1,8 @@
 import React from 'react'
 import { BrowserRouter, Routes, Route } from 'react-router-dom'
-import Login from './scenes/Login'
-import Home from './scenes/Home'
-import Resume from './scenes/Resume/Resume'
+import Login from 'scenes/login'
+import Home from 'scenes/home'
+import Resume from 'scenes/resume'
```
Comment (thaoho21294) [inline src/App.js:5]: Look like you forgot to update the folder name from `Resume` to `resume`.

File: src/components/editor/BlockButton.js [hunk 2]
```diff
@@ -1,12 +1,14 @@
...
+import Icon from 'components/icon'
 import { isBlockActive, toggleBlock } from 'utils/editor'
-import { TEXT_ALIGN_TYPES } from '../../../constants'
+import { TEXT_ALIGN_TYPES } from '../../constants'
 
 const BlockButton = ({ format, icon }) => {
-  const editor = useSlate()
+  const editor = 
+  
+  useSlate()
```
Comment (thaoho21294) [inline src/components/editor/BlockButton.js]: typing error? should be inline

**Comment by thaoho21294** [comment]
Suggest reset to 1 commit with a message: Restructure folders

File: src/components/editor/BlockButton.js (hunk 2 above)
Comment (tungle251) [reply src/components/editor/BlockButton.js]: yes, fixed

**Review by thaoho21294 — APPROVED** [review]

**Pull request merged**